.github/actions/spell-check/expect.txt

```diff
@@ -2175,8 +2180,8 @@ vsonline
 vstemplate
 VSTHRD
 VSTT
-vtable
 VTABLE
+vtable
```

Comment:
To fix the spellchecking error, just remove the uppercase "VTABLE" and keep the lowercase one in the file.

Reply:
I actually am not sure where `VTABLE` is even used because I don't have that :D

Reply:
I think it has been caused by the reordering of these.
Still, please delete `VTABLE` to fix the spellchecker issues.

Reply:
Done just now @jaimecbernardo